service/history/execution/context.go

```diff
@@ -415,14 +415,14 @@ func (c *contextImpl) CreateWorkflowExecution(
 
 	_, err := c.createWorkflowExecutionWithRetry(ctx, createRequest)
 	if err != nil {
+		if c.isPersistenceTimeoutError(err) {
```

Comment:
Will it be a problem to notify task regardless error type?

Reply:
No, you can notify task processor as many times as you want. It will only trigger a DB load.